18/README.md

```diff
@@ -241,6 +244,47 @@ Below is an example _package.json_ file with the _main_ attribute and _start_ sc
 #### Note:
 `oc rsync` is only available in versions 3.1+ of OpenShift.
 
+## init-wrapper
+
+init-wrapper script is located on `/usr/libexec/s2i/init-wrapper` and is used for preventing below circumstances:
+
+- Proper signal handling and propagation, as Node.js was not designed to run as PID 1.
+- Reaping zombie child processes
+- Avoiding use of npm, more information on [reference architecture](https://github.com/nodeshift/nodejs-reference-architecture/blob/e4c4dc1fd20c2cac392e862859aaad27f85d504f/docs/development/building-good-containers.md#avoiding-using-npm-to-start-application)
```

Comment:
```suggestion
Avoiding use of npm, there is more information on why you want to avoid that in the [Node.js reference architecture](https://github.com/nodeshift/nodejs-reference-architecture/blob/e4c4dc1fd20c2cac392e862859aaad27f85d504f/docs/development/building-good-containers.md#avoiding-using-npm-to-start-application)
```